### ISSUE TITLE
Dropbox / Google Drive are not available

### ISSUE THREAD

**Issue by Emilia-Re** [KMnP/vpt#74]
Dropbox / Google Drive are not available,please update the link.

**Comment by Yiming-M** [comment]
I am also interested to see what's the optimal setup for hyper-parameters like the learning rate, weight decay, and scheduling scheme.

**Comment by bfshi** [comment]
Here's a full copy of FGVC data (including the train/test split json files) prepared according to the original instructions by the authors, if it helps:

https://berkeley.box.com/shared/static/kt2sla80lmrmldiylltwva82tddr2jc3.tar

**Comment by LaughAwayZHY** [comment]
> Here's a full copy of FGVC data (including the train/test split json files) prepared according to the original instructions by the authors, if it helps:
> 
> https://berkeley.box.com/shared/static/kt2sla80lmrmldiylltwva82tddr2jc3.tar

Hello, do you have the link to the VTAB-1k dataset and the hyperparameter values?
